ConnectionJustForToday/ConnectionJustForToday/HelperFiles/JFTService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  JFTService.swift
+//  ConnectionJustForToday
+//
+//  Created by Matthew Hill on 4/4/23.
+//
+
+import Foundation
+import Collections
```

Comment:
if you're no longer using an ordered dictionary, you do not need to import collections (which, I agree that you should not be using an ordered dictionary)